build/Dockerfile.DiskVirt

```diff
@@ -0,0 +1,34 @@
+FROM registry.access.redhat.com/ubi9/ubi-minimal AS taskBuilder
+RUN microdnf install -y tar gzip && microdnf clean all
+ENV TASK_NAME=disk-virt-sysprep \
+    GOFLAGS="-mod=vendor" \
+    GO111MODULE=on
+WORKDIR /src/${TASK_NAME}
+RUN curl -L https://go.dev/dl/go1.20.2.linux-amd64.tar.gz | tar -C /usr/local -xzf -
+ENV PATH=$PATH:/usr/local/go/bin
+
+COPY modules/. /usr/local/go/src
+RUN mkdir /out
+
+RUN for TASK_NAME in "disk-virt-customize" "disk-virt-sysprep"; do \
+        cd /usr/local/go/src/$TASK_NAME/; \
+        CGO_ENABLED=0 GOOS=linux go build -o /out/${TASK_NAME} cmd/${TASK_NAME}/main.go; \
+    done
+
+FROM quay.io/kubevirt/libguestfs-tools:v0.59.0
+ENV TASK_NAME=disk-virt
+ENV USER_UID=1001 \
+    USER_NAME=${TASK_NAME} \
+    HOME=/home/${TASK_NAME}
+
+USER 0
```

Comment:
This is overriden in L34?

Reply:
Yes, the libguestfs does not allow to do user modifications without switching to root. Then on line 34 it is switched back to regular user